draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -120,7 +124,20 @@ in Section 7.1 of {{HTMLSpec}}) has its own ICE agent.
 ICE Candidate Gathering {#gathering}
 ----------------------------
 
-For any host candidate gathered by an ICE agent as part of {{RFC8445}}, Section 5.1.1, the candidate is handled as follows:
+This section outlines how mDNS should be used by ICE agents to conceal local
+IP addresses.
+
+For each host candidate gathered by an ICE agent as part of the gathering
+process described in {{RFC8445}}, Section 5.1.1, the candidate is handled as
+described below. However, in certain cases, the ICE agent may decide that
+it does not need to conceal a candidate IP, and this processing may be skipped
+for that candidate.
+
+This may be because the ICE agent has a priori knowledge that the candidate
+address is in fact public, or because the agent has made a policy decision to
+not conceal certain types of IP addresses (e.g., those with built-in privacy
+protections) as a calculated choice to improve connectivity. This topic is
+discussed further in {#privacy} below.
```

Comment:
There is something missing as below is an enumeration of steps.
Maybe it would read better if there was a step 0 which is to decide whether to conceal a candidate IP or not.

Discussion of this particular step could be done after the description of the steps.

Reply:
Something like:

> 0. Check whether this IP address satisfies the ICE agent's policy for address privacy. If so, expose the candidate and abort this process.

(later)

> ICE agents may decide that certain local IP addresses are safe to expose. 
> This may be because the ICE agent has a priori knowledge that the
> address is in fact public, or because the agent has made a policy decision to
> not conceal certain types of IP addresses (e.g., those with built-in privacy
> protections) as a calculated choice to improve connectivity. This topic is
> discussed further in {#privacy} below.

Reply:
@youennf, does this match what you were thinking? Any other outstanding concerns?

Reply:
Yes, sounds good.

Reply:
Done.